app/src/components/staking/DelegationsOverview.vue

```diff
@@ -11,13 +11,37 @@
           alt="geometric placeholder shapes"
         />
       </div>
-      <div v-else-if="delegations.length > 0">
+      <div v-else-if="delegations.length > 0 || stakedBalance.total > 0">
         <h1>Your Stake</h1>
+        <Bar
```

Comment:
for this one, we already have a "no validator" message. let's use that instead.

Reply:
and the button / call to action?